docs/tsc/meetings/2019-03-04.md

```diff
@@ -0,0 +1,92 @@
+  * [X] Thanh Ha - Academy Software Foundation / Linux Foundation
+  * [ ] Patrick Hodoul (_TSC_) - Autodesk
+  * [X] John Mertic - Academy Software Foundation / Linux Foundation
+  * [X] Carl Rand (_TSC_) - Foundry
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+
+# **OCIO TSC Meeting Notes**
+
+* Feature preview tag / Early 2.0 release
+    - Decision to NOT do either a preview tag or early 2.0 release
+        - Doug: As established in the initial 2.0 proposal, this is not feasible.
+        - The plan remains to release a complete feature set when the full v2 roadmap is completed.
+    - Community concern is that the v2 release schedule is unknown, and time between OCIO releases is long.
+        - This can make planning for software vendors challenging, but preliminary implementation of the v2 interface is possible now from master for testing.
+        - Autodesk discussing an official roadmap and insight on a potential release window. Stay tuned.
+        - Long PR review cycles contribute to development slowdowns:
+            - Most PRs are merged with the 2 week single reviewer clause.
+            - Larry: If current merge policies are a roadblock to AD, can they be revised?
+                - Proposed rotating review duty for a TSC member to do initial PR reviews as they are submitted for their assigned week
+            - This needs to be improved if it's blocking.
+            - Doug: Feature interdependency part of the challenge:
+                - PRs are becoming smaller, more focused, which should help with review accessability
+            - **TODO**: Doug/Patrick to review and report back on what project management changes could help them
+    - Larry: Should we have a known commit for testing a given feature (i.e. GPU renderer) for internal testing purposes.
+        - Doug: Testing off of master at any given time is stable, and ideal.
+
+* ASWF/LF CI update from TAC meeting
+    - Thanh presented comparison of Circle CI and current Jenkins infrastructure:
+        - Overall, TAC impression is that Circle seems like a better choice, but pricing and logistics needs further investigation.
+        - Circle supports local CI, allowing contributors to test CI *before* submitting a PR.
+        - Supports GPUs when hosted on our servers, but pricing may be a concern.
+        - Proposed AppVeyor for Windows support for now.
+    - OpenVDB reports that Circle significantly faster than Travis
+    - Larry: Circle has better, faster CPUs than Travis
+    - **TODO**: Once ASWF/LF commit to CI choice, Mark/Sean to investigate transition to new CI infastructure
+        - If OpenVDB adopts new system first, we can reference their setup.
+
+* Moving OCIO to ASWF Git organization
+    - Legal approvals for OCIO domain tranfer are complete!
+    - The GitHub account move should be fairly straightforward, but may bring down CI builds for a day or two diring transition.
```

Comment:
*Comment:* The two repos must coexist for a while as many pull requests already exist and we also have several internal code reviews.